.editorconfig

```
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Ironically, there is no newline following this line :sweat_smile: 
```suggestion
insert_final_newline = true

```